src/state/slices/opportunitiesSlice/thunks.ts

```diff
@@ -203,7 +213,25 @@ export const fetchAllLpOpportunitiesUserdata = async (
           ),
         ),
     ),
-  )
+    store.dispatch(
+      getOpportunitiesUserData.initiate(
+        {
+          accountId,
+          defiType: DefiType.LiquidityPool,
+          defiProvider: DefiProvider.Osmosis,
+          opportunityType: DefiType.LiquidityPool,
+        },
+        {
+          forceRefetch: false,
+          ...options,
+        },
+      ),
+    ),
+  ]
+
+  for (const userDataPromise of userDataPromises) {
+    await userDataPromise
+  }
```

Comment:
Can we do this guy in parallel, or does it need to be synchronous? I assume that's what defi was referring to above.

```suggestion
  await Promise.allSettled(userDataPromises)
```